rest/models.go

```diff
@@ -2,9 +2,13 @@ package rest
 
 import (
 	"github.com/akitasoftware/akita-libs/akid"
+	"github.com/akitasoftware/akita-libs/api_schema"
 )
 
+// TODO: shouldn't this be in akita-cli/api_schema?
```

Comment:
Did you intend to address this?

Reply:
No, not really, I didn't take the time to investigate why this was the only model here.